src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -107,9 +110,19 @@ export default Vue.extend<Data, Methods, unknown, unknown>({
             workflowRunStatus: null,
             error: "",
             defaultMessage: "",
+            polling: null
         };
     },
     methods: {
+        startPolling() {
+            if(!this.polling) {
+                this.polling = setInterval(() => this.getWorkflowRunStatus(this.selectedWorkflowKey), 1500);
+            }
+        },
+        stopPolling() {
```

Comment:
Should we stop polling if component is unmounted?

Reply:
Could stop polling in `beforeUnmount` for the change tab case. Timer should be cleared anyway if navigate to a completely different page.